workloads/k8s-io/run.sh

```
@@ -33,7 +33,11 @@ set +e
 
 JOB_START=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
 
-cmd="timeout ${TEST_TIMEOUT} ./${K8S_IO_FILENAME} -config ${CONFIG}"
+RESOLVED_CONFIG=$(mktemp -t k8s-io-config-XXXX.yaml)
+envsubst < "${CONFIG}" > "${RESOLVED_CONFIG}"
+trap "rm -f ${RESOLVED_CONFIG}" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)